### PR TITLE
parser for kinto command

### PR DESCRIPTION
**Pull request by ayusharma**
Kinto --version command

### CODE REVIEW

File: kinto/__main__.py [hunk 1]
```diff
@@ -30,6 +30,8 @@ def main(args=None):
                         dest='backend',
                         required=False,
                         default=None)
+    parser.add_argument('--version', action='version', version=__version__,
+                            help='Print the Kinto version and exit.')
```
Comment (almet) [inline kinto/__main__.py]: flake8 reports that ``kinto/__main__.py:34:29: E127 continuation line over-indented for visual indent``

**Comment by almet** [comment]
Thanks, that looks great, you just need to address the little nitpick and we're good to merge!

File: kinto/__main__.py [hunk 2]
```diff
@@ -30,6 +30,8 @@ def main(args=None):
                         dest='backend',
                         required=False,
                         default=None)
+    parser.add_argument('--version', action='version', version=__version__,
```
Comment (Natim) [inline kinto/__main__.py]: Should we also support the sort '-v' option?

    parser.add_argument('-v', '--version', action='version', ...

Comment (almet) [inline kinto/__main__.py]: I believe this will do it by default. In which case it could be good to define it explicitely here, you're right.

**Comment by Natim** [comment]
Fixed and merged in 55de173

**Comment by Natim** [comment]
Thank you for that :+1:

**Pull request merged**